Weather/Services/APIService.swift

```diff
@@ -12,6 +12,11 @@ struct APIService {
     static let baseURL = "https://api.openweathermap.org/data/2.5/"
     
     static func getURLFor(latitude: Double, longitude: Double) -> String {
-        return "\(baseURL)onecall?lat=\(latitude)&lon=\(longitude)&exclude=minutely&appid=\(apiKey)&units=metric"
+        let languageCode = Locale.current.languageCode
+        if languageCode == "ru" {
```

Comment:
I would say that mostly the US uses imperial system. Other countries use metric system. I would suggest you to allow users to select system in some settings. Otherwise people from the UK who also speak English will receive imprerial data instead of metric that they use